### DISCUSSION TITLE
Elfutils for Mac OS Ventura 13.3.1

### DISCUSSION THREAD

**Discussion by apurvakhatri** [bloomberg/pystack#109] [General]
In order to get the dependencies (libdw & libelf) on my system I went ahead with installing elfutils.
I am unable to install elfutils using homebrew. Below is the error:

Command: brew install elfutils
Logs:
==> Downloading https://formulae.brew.sh/api/formula.jws.json
#=#=#                                                                          
==> Downloading https://formulae.brew.sh/api/cask.jws.json
#=#=#                                                                          
elfutils: Linux is required for this software.
Error: elfutils: An unsatisfied requirement failed this build.

Eventually, I get the below error when trying to execute "python3 -m pip install -e .
"

Error:         cmd_obj.run()
      error: command '/usr/bin/clang' failed with exit code 1
      [end of output]

Note: libelf is available with homebrew & installing it was a success. It is the dependency libdw which is causing the trouble.

I was looking for suggestions on how to get the dependencies on my system. Any suggestions would be highly appreciated. Thank you!

**Comment by pablogsal** [comment]
Hi @apurvakhatri and thanks for opening this discussion. Unfortunately `pystack` doesn't work on macOS as it only works on Linux so even if you can get the dependencies there, it won't work correctly. Apologies for the confusion, we will ensure that the error message is better in the future when trying to install it on macOS

**Reply by dandiep** [reply]
Might be good to add supported os's to the home page as well. It took me some digging to find this thread.

**Reply by pablogsal** [reply]
👍
